collections/guides/encrypt_us_bank_accounts/index.md

```diff
@@ -35,14 +36,16 @@ If you'd like to follow along with this guide jump right into the code - <a href
 
 ## The existing application
 
-Below is an existing nodejs application that is currently storing bank data in memory, this will be used as our foundation as we update to more securely store the bank data. 
+Below is an existing Node.js application that is currently storing bank data in memory, this will be used as our foundation as we update to more securely store the bank data. 
+
 
 _(Note: this is only to simplify the guide, the same process will work when storing in a database)_.
 
 ### package.json
 Our simple `package.json`, only has two dependencies to start. [express](https://www.npmjs.com/package/express) allows us to quickly create an HTTP API, and [nodemon](https://www.npmjs.com/package/nodemon) allows us to have our nodejs application restart automatically.
 
-```js
+```json

```

Comment:
Be careful with the styling here - I know this is the "correct" type of code block to use for this content, but I noticed in the guides that if I tried to classify snippets as `json` or `bash` (anything other than js), then the background changed to white and the styling was thrown off from all the other code samples. We should eventually add some custom css to apply the same styles to other code block types.

Reply:
This is a good call. I looked at this the other day and covered a few of them, and I just didn't have enough time to make it work across them all. 😬. I'll pull it down and verify

Reply:
![image](https://user-images.githubusercontent.com/151590/148884575-654647ed-0154-4375-a095-e04f86686e30.png)


Double checked this and it looks like this.. will want to update this before we move on. @djejaquino do we have any styling we can use here for json or change back to js for now?

Reply:
assuming that isn't what we want i guess.

Reply:
updated this 
![image](https://user-images.githubusercontent.com/151590/149064131-d8cce2f7-c677-463f-9c82-66d76cc1fda9.png)

Reply:
I am good with whatever looks good. Just suggested `json` as I never had problems with it